src/main/kotlin/no/nav/familie/ef/sak/service/PersisterGrunnlagsdataService.kt

```diff
@@ -40,13 +41,19 @@ class PersisterGrunnlagsdataService(private val pdlClient: PdlClient,
             MDC.setContextMap(context)
             grunnlagsdataRepository
                     .finnBehandlingerSomManglerGrunnlagsdata()
-                    .forEach {
-                        logger.info("Lagrer grunnlagsdata for $it")
+                    .forEach { (behandlingId, status) ->
+                        logger.info("Lagrer grunnlagsdata for $behandlingId")
                         try {
-                            opprettGrunnlagsdata(it)
+                            val grunnlagsdata = hentGrunnlagsdataFraRegister(behandlingId)
+                            val type = if (BehandlingStatus.valueOf(status).behandlingErLåstForVidereRedigering())
+                                GrunnlagsdataType.BLANKETT_ETTER_FERDIGSTILLING else GrunnlagsdataType.V1
+
+                            grunnlagsdataRepository.insert(Grunnlagsdata(behandlingId = behandlingId,
+                                                                         data = grunnlagsdata,
+                                                                         type = type))
```

Comment:
Her setter vi BLANKETT_ETTER_FERDIGSTILLING på de som allerede er låst for videre redigering